packages/amplify-console-hosting/utils/path-manager.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
```

Comment:
Why do you need to re-mplement this file and not use `context.amplify` instead?

Reply:
Caused I wants template path and other paths that are not in context.amplify to be reused across packages. Just in consistence, wrap all native path-manager API I used in this utils code as well